src/HomePage/HomePage.js

```diff
@@ -6,18 +9,43 @@ import Container4 from './Container_4'
 import Container5 from './Container_5'
 import Container6 from './Container_6'
 
+import { HomePagePics, HomePageCaptions }  from '../pictures/HomePagePics'
 import './HomePage.css'
 
 export default function HomePage() {
+
+//pictures imported
+const pictures = HomePagePics;
+
+//display modal
+const [modalDisplay, changeModalOpen] = useState(false)
+
+//changing currently displayed polaroid on page and in modal
+const [display,toDisplay] = useState(0)
+const changeDisplay = (number) => {
+    if(number > 0) {
+        if(display === pictures.length - 1) toDisplay(0)
+        else toDisplay(display + 1)
+    } 
+    else {
+        if(display === 0) toDisplay(pictures.length - 1)
+        else toDisplay(display - 1)
+    }
+}
     return (
         <div className = "HomePage">
+            <Modal pictures = {HomePagePics} captions = {HomePageCaptions} modalDisplay = {modalDisplay} changeModalOpen = {changeModalOpen}
```

Comment:
I need to see the `Modal` file but it's kind of sus that you're passing in things that you imported into this file as properties. Can't you import it into the `Modal` container?

Reply:
It's so I can use the modal in different areas - I was planning to use it in the about page as well